src/message_interface/message_handler.rs

```diff
@@ -118,12 +118,23 @@ impl WalletMessageHandler {
         log::debug!("Message: {:?}", message);
 
         let response: Result<Response> = match message {
-            Message::CreateAccount(account) => {
-                convert_async_panics(|| async { self.create_account(&account).await }).await
+            Message::CreateAccount { alias } => {
+                convert_async_panics(|| async { self.create_account(alias).await }).await
```

Comment:
That really doesn't require a change in the bindings ?

Reply:
No